### DISCUSSION TITLE
SEI formation

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3107] [Q&A]
How does pybamm account for the initial SEI formation in the cell and how does it reflect in the first cycle efficiency.
Does pybamm (by default) account for the initial SEI formation in the cell or do we have to give SEI parameters and include aging mechanisms to include initial SEI formation occurring in the first couple of cycles?
And when modelling aging mechanisms in pybamm does the initial formed SEI is already included?

**Comment by valentinsulzer** [comment]
PyBaMM doesn't model the initial SEI formation process, you have to specify an initial SEI thickness to account for the initial SEI that has already been formed

**Reply by kumaryash7** [reply]
Thank you!
Can you please elaborate on the inner and the outer sei layer meaning and reference used in the parameter sets.

**Reply by shubhamrawat2209** [reply]
how to get inner and outer initial layer thickness value for Okane2022 parameters?

**Comment by kumaryash7** [comment]
> PyBaMM doesn't model the initial SEI formation process, you have to specify an initial SEI thickness to account for the initial SEI that has already been formed
And also do I have to include aging option in the model to incorporate initial SEI layer even if I don't want to study aging effects. I only want to include the initial SEI growth. 
What is the model option to use and the parameter required for incorporating only the initial SEI growth.
